openslides/config/views.py

```diff
@@ -100,3 +101,22 @@ def form_valid(self, form):
             config[key] = form.cleaned_data[key]
         messages.success(self.request, _('%s settings successfully saved.') % _(self.config_collection.title))
         return super(ConfigView, self).form_valid(form)
+
+
+class ConfigViewSet(viewsets.ViewSet):
```

Comment:
Why not adding a flag to every config variable `required_permission_to_see`. Default is None which means that there is no special permission required to see this variable. At the moment there is no variable which has secret content but we (or plugin developers) can add a special permission to variables later on.